packages/bitcore-client/bin/multi-sig/wallet-derive-address

```diff
@@ -2,21 +2,32 @@
 
 const fs = require('fs');
 const bitcoreLib = require('bitcore-lib');
+const bitcoreLibCash = require('bitcore-lib-cash');
```

Comment:
These requires weren't working for me on a fresh install. I think we need to refer to crypto wallet core instead. 

```
const CWC = require('crypto-wallet-core');
const bitcoreLibs = {
  BTC: CWC.BitcoreLib,
  DOGE: CWC.BitcoreLibDoge,
  LTC: CWC.BitcoreLibLtc,
  BCH: CWC.BitcoreLibCash
};
```